src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -201,15 +201,38 @@ def _get_variance(self, timestep, prev_timestep):
 
     # Copied from diffusers.schedulers.scheduling_ddpm.DDPMScheduler._threshold_sample
     def _threshold_sample(self, sample: torch.FloatTensor) -> torch.FloatTensor:
-        # Dynamic thresholding in https://arxiv.org/abs/2205.11487
-        dynamic_max_val = (
-            sample.flatten(1)
-            .abs()
-            .quantile(self.config.dynamic_thresholding_ratio, dim=1)
-            .clamp_min(self.config.sample_max_value)
```

Comment:
we do not want to clamp the min to the max value. we want to min clamp to one and max clamp to the config value.

min clamping to one is equivalent to the standard [-1, 1] clamp. See the rewritten function for example. Also see the lucidrains implementation which while does not max clamp, does min clamp to 1 https://github.com/lucidrains/imagen-pytorch/blob/76b295086c345ccb5adb946d607848361b4a2189/imagen_pytorch/imagen_pytorch.py#L2092